idaes_examples/notebooks/docs/power_gen/ngcc/ngcc.ipynb

```diff
@@ -107,7 +114,8 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": null,
```

Comment:
I noticed that none of the cells have been executed in this notebook. All the other example notebooks I've seen have their cells executed, but I'm not sure if this ultimately matters or not.

Reply:
@andrewlee94 Should the cells be executed before this is approved?

Reply:
That is more a question for @lbianchi-lbl. I do not know exactly how things get run and generated.